ReactWindows/ReactNative/Views/DatePicker/ReactCalendarDatePickerManager.cs

```diff
@@ -0,0 +1,232 @@
+using System;
+using Windows.UI.Xaml.Controls;
+using ReactNative.UIManager;
+using ReactNative.UIManager.Annotations;
+using ReactNative.UIManager.Events;
+using Newtonsoft.Json.Linq;
+
+namespace ReactNative.Views.DatePicker
+{
+    /// <summary>
+    /// A view manager responsible for rendering a CalendarDatePicker.
+    /// </summary>
+    public class ReactCalendarDatePickerManager : BaseViewManager<CalendarDatePicker, ReactCalendarDatePickerShadowNode>
+    {
+        /// <summary>
+        /// The name of the view manager.
+        /// </summary>
+        public override string Name
+        {
+            get
+            {
+                return "CalendarDatePickerWindows";
+            }
+        }
+
+        /// <summary>
+        /// Sets the value of the picker.
+        /// </summary>
+        /// <param name="view">The picker view element.</param>
+        /// <param name="date">The new value.</param>
+        [ReactProp("date")]
+        public void SetDate(CalendarDatePicker view, DateTime? date)
+        {
+            view.Date = date ?? null;
+        }
+
+
+        /// <summary>
+        /// Sets the maximum allowed date of the picker.
+        /// </summary>
+        /// <param name="view">The picker view element.</param>
+        /// <param name="date">The value to set as maximum.</param>
+        [ReactProp("maxDate")]
+        public void SetMaxDate(CalendarDatePicker view, DateTime? date)
+        {
+            if (date.HasValue)
```

Comment:
Do you want to clear out the `MaxDate` if the prop value is unset or reset to null?

Reply:
Good idea. Will do.